core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigrationActor.scala

```diff
@@ -730,39 +830,37 @@
   case object RunMigration extends Message
   case object RefreshTransferJobs extends Message
 
-  // ehigham: I've included this actor implementation as it's helped inform how to refresh
-  // the status of storage transfer jobs. While it's as yet untested, this can be plumbed-in
-  // and integration tested as part of CA-1132.
-  // todo: finalise design and test in CA-1132
+
   def apply(pollingInterval: FiniteDuration,
             dataSource: SlickDataSource,
             googleProjectToBill: GoogleProject,
             workspaceService: WorkspaceService,
             storageService: GoogleStorageService[IO],
-            storageTransferService: GoogleStorageTransferService[IO])
+            storageTransferService: GoogleStorageTransferService[IO],
+            sam: SamDAO,
```

Comment:
Prefer rename `sam` -> `samDAO`.  Mentioned this elsewhere too, so I won't keep commenting on it, but I would prefer this name change across the board.